<launch>
  <arg name="serial_number" />
  <arg name="camera_name" />

  <group ns="$(arg camera_name)">

    <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager" args="manager" />
    <node pkg="nodelet" type="nodelet" name="camera_nodelet"
          args="load pointgrey_camera_driver/PointGreyCameraNodelet camera_nodelet_manager" >

      <param name="frame_id" value="$(arg camera_name)" />
      <param name="serial" value="$(arg serial_number)" />
      <param name="frame_rate" value="30" />
      <param name="video_mode" value="format7_mode0" /> <!--Max res 1288 x 964 -->
    </node>

  </group>

</launch>
